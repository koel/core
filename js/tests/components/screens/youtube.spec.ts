import Component from '@/components/screens/youtube.vue'
import { shallow } from '@/tests/adapter'

describe('components/screens/youtube', () => {
  it('renders properly', () => {
    expect(shallow(Component)).toMatchSnapshot()
  })
})
