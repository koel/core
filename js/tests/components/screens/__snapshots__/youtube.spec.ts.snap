// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/screens/youtube renders properly 1`] = `
<section id="youtubeWrapper">
  <h1 class="heading"><span>YouTube Video</span></h1>
  <div id="player">
    <p class="none">Your YouTube video will be played here.<br>
      You can start a video playback from the right sidebar. When a song is playing, that is.<br>
      It might also be worth noting that video’s volume, progress and such are controlled from within
      the video itself, and not via Koel’s controls.</p>
  </div>
</section>
`;
